01-Login/README.md

```diff
@@ -1,84 +1,705 @@
+You can either install the library locally in your application or load it from CDN.
+
+### Loading via dependencies
+
+Install auth0.js using npm or yarn.
 
 ```bash
-npm install
+# installation with npm
+npm install --save auth0-js
+
+# installation with yarn
+yarn add auth0-js
+```
+
+Once you install auth0.js, add it to your build system or bring it in to your project with a script tag.
+
+```html
+<script type="text/javascript" src="node_modules/auth0-js/build/auth0.js"></script>
 ```
 
-### Configuration
+### Loading it from CDN
+ 
+If you do not want to use a package manager, you can retrieve auth0.js from Auth0's CDN.
+
+```html
+<script src="${auth0js_url}"></script>
+```
+
+### Authentication with Auth0
+
+[Universal Login](/hosted-pages/login) is the easiest way to set up authentication in your application. We recommend using it for the best experience, best security and the fullest array of features. This guide will use it to provide a way for your users to log in to your ${library} application.
+
+::: note
+You can also embed the login dialog directly in your application using the [Lock widget](/lock). If you use this method, some features, such as single sign-on, will not be accessible. 
+<% if (typeof embeddedLoginLink == 'string') { %>
+To learn how to embed the Lock widget in your application, follow the [Embedded Login sample](${embeddedLoginLink}).
+<% } %>
+:::
+
+When a user logs in, Auth0 returns three items:
+* `access_token`: to learn more, see the [Access Token documentation](/tokens/access-tokens)
+* `id_token`: to learn more, see the [ID Token documentation](/tokens/id-tokens)
+* `expires_in`: the number of seconds before the Access Token expires
+
+You can use these items in your application to set up and manage authentication. 
 
-The project needs to be configured with your Auth0 domain and client ID in order for the authentication flow to work.
+### Create an Authentication Service
 
-To do this, first copy `auth_config.sample.json` into a new file in the same folder called `auth_config.json`, and replace the values within with your own Auth0 application credentials:
+The best way to manage and coordinate the tasks necessary for user authentication is to create a reusable service. With the service in place, you will be able to call its methods throughout your application. The name for it is at your discretion, but in these examples it will be called `AuthService` and the filename will be `authService.js`. An instance of the `WebAuth` object from **auth0.js** can be created in the service.
+
+Create a service and instantiate `auth0.WebAuth`. Provide a method called `login` which calls the `authorize` method from auth0.js.
+
+```js
+// src/auth/authService.js
+
+import auth0 from 'auth0-js';
+import EventEmitter from 'events';
+import authConfig from '../../auth_config.json';
+
+const webAuth = new auth0.WebAuth({
+  domain: authConfig.domain,
+  redirectUri: `<%= "${window.location.origin}" %>/callback`,
```

Comment:
The `<%=` escape characters are a docs thing and would be rendered out in GitHub, so these should be removed.

This line would become:

```suggestion
  redirectUri: `${window.location.origin}/callback`,
```